debian/changelog

```diff
@@ -1,3 +1,10 @@
+mobile-broadband-provider-info (20220809-ubports3) xenial; urgency=medium
```

Comment:
Next time, please reset `-ubports3` part to `-0ubports1`. But for this time the current one will do.